frontend/tests/jest/utils.tsx

```diff
@@ -157,3 +163,30 @@ export const getProductWithWarranty = (
 
    return product;
 };
+
+export const getReviewsResponse = (
+   type: 'filled' | 'empty' = 'filled'
+): ProductReviewData => {
+   let reviews: ProductReview[] = mockedReviews;
+
+   if (type === 'empty') {
+      reviews = [];
+   }
+
+   return {
+      reviews: reviews,
+      count: reviews.length,
+      average:
+         reviews.length == 0
+            ? 0 // @ts-ignore
+            : reviews.reduce((total, next) => total + next.rating, 0) /
```

Comment:
I would rather we not use the ts-ignore in general if possible. It feels sloppy and can ignore more errors than the one you were trying to ignore. A simple workaround can make this pass the type check, though. This suggestion is a very simple but maybe slightly wrong way to do it, since it will create a 0-rating review that might not be right, but that might be just fine in a test that already ignores that problem. It may be the case that reviews can't actually have a 0 rating. We'd have to check the api that returns these and see if a null/undefined rating is possible, but if not, we can adjust the type. If we must support reviews without ratings and if we must handle that case in the test, then the most accurate way to handle it is to filter out reviews without ratings before this calculation. But I think we would be close enough to just accept the suggestion and move on.
```suggestion
            ? 0
            : reviews.reduce((total, next) => total + (next.rating || 0), 0) /
```